pkg/manager/manager.go

```diff
@@ -74,8 +74,8 @@ type Manager interface {
 	// GetCache returns a cache.Cache
 	GetCache() cache.Cache
 
-	// GetRecorder returns a new EventRecorder for the provided name
-	GetRecorder(name string) record.EventRecorder
+	// GetEventRecorderFor returns a new EventRecorder for the provided name
+	GetEventRecorderFor(name string) record.EventRecorder
```

Comment:
This is a breaking change. Was that intentional? If not, can we add the `GetRecorder` method back and deprecate it?

Reply:
That was marked as breaking.  Did it make it into a stable branch?  We should do a patch to the stable branch to restore it.

Reply:
Doesn't look like it made it into a stable branch (it's not on release 0.1.10).  Master is intended to have breaking changes, and yes, this was intended to be breaking (it brings manager in line with other things that provide a recorder).

Reply:
Oops, I missed the breaking mark (I wasn't aware of the meaning of ⚠️), so please ignore my comment. Thanks for explaining @DirectXMan12!